wordpress-mobile/WordPress-iOS
# [Domain Management] Add My Domains to Me Screen

Refs #21424
Refs #21425

This PR adds the "My Domains" tab to "Me" screen and tapping on the button will open the empty view controller.

# Testing Steps
1. Install & Login Jetpack App
2. `RemoteFeatureFlag` `domainManagement` default value on line 63. needs to be set to true.
3. Go to "Me"
4. Verify if a "My Domains" tab appears under the new "Products" section. (The tab will be made shorter to match the designs on a separate PR).
5. Tap on the button and verify if a new screen is presented.
